vendor/k8s.io/kube-openapi/pkg/util/trie.go

```diff
@@ -14,17 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package trie
```

Comment:
without any import this should definitely go to k8s.io/util. Here – used by other components – it is not at the right place.